bcipy/helpers/copy_phrase_wrapper.py

```diff
@@ -177,27 +183,24 @@ def evaluate_eeg_evidence(self, raw_data: np.array,
         """
         letters, times, target_info = self.letter_info(triggers, target_info)
 
-        # Remove 60hz noise with a notch filter
-        notch_filter_data = notch.notch_filter(
-            raw_data, self.sampling_rate,
-            frequency_to_remove=self.notch_filter_frequency)
-
-        # bandpass filter from 2-45hz
-        filtered_data = bandpass.butter_bandpass_filter(
-            notch_filter_data,
-            self.filter_low,
-            self.filter_high,
-            self.sampling_rate,
-            order=self.filter_order)
-
-        # downsample
-        data = downsample.downsample(
-            filtered_data, factor=self.downsample_rate)
-        x, _, _, _ = trial_reshaper(target_info, times, data,
-                                    fs=self.sampling_rate,
-                                    k=self.downsample_rate, mode=self.mode,
-                                    channel_map=self.channel_map,
-                                    trial_length=window_length)
+        default_transform = get_default_transform(
+            sample_rate_hz=self.sampling_rate,
+            notch_freq_hz=self.notch_filter_frequency,
+            bandpass_low=self.filter_low,
+            bandpass_high=self.filter_high,
+            bandpass_order=self.filter_order,
+            downsample_factor=self.downsample_rate,
+        )
+        data, self.sampling_rate = default_transform(raw_data, self.sampling_rate)
+
+        x, _ = data_reshaper(input_data_type=self.signal_model.input_data_type,
```

Comment:
small change... let's assign a better label to this? `trials` or something equally descriptive works for me

Reply:
I went with `data` for now, since it will sometimes be trials and sometimes inquiries